SwiftLint/swiftlint+frameworks.yml

```diff
@@ -7,3 +7,7 @@
 
 #opt_in_rules:
 #  - explicit_top_level_acl
+#  - missing_docs
+
+#missing_docs:
+#  warning: open, public #, internal TODO: enable check for internal at some point
```

Comment:
we need to discuss this. It will not work as-is the logic has been improved.
But adding custom configuration for a specific rule might be out the question.

Reply:
Just checked, the correct approach would be to configure the rules in the basic 'general' swiftlint.yml file.
On the swiftlint+mystuff.yml file only opt_in or disable the rules.
 
I'm clong this PR as this logic has been integrated in the other one.

Reply:
That's an improvement already but it might very well be that the frameworks intentionally want a different configuration on certain rules than the apps.

Reply:
@proth this rule with this config (without internal) adds 70 warnings on the Sync framework only. The integration of Swiftlint which is now fixed adds ~100 warnings and few compilation errors... on top of the missing_docs warnings.

Let's talk later, once you are done fixing **all** the warnings, for **all** projects and **all** frameworks about "what rule configuration should we have for each projects"

Reply:
Sure. That's why internal was commented inside the comment.
I just wanted to create awareness that it would be good to have a way to enforce some harder rules or rules with different configurations for the frameworks. I thought it makes sense to mention now since you're redoing the bash script.